pdc/apps/release/views.py

```diff
@@ -333,7 +333,7 @@ class ReleaseViewSet(ChangeSetCreateModelMixin,
     """
     queryset = models.Release.objects \
                      .select_related('product_version', 'release_type', 'base_product') \
-                     .prefetch_related('compose_set').order_by('id')
+                     .order_by('id')
```

Comment:
Why don't we also remove select_related function?

Reply:
I don't see any issues with that one.

Reply:
ok